templates/Web/Projects/ReactDefault/package_postaction.json

```diff
@@ -0,0 +1,7 @@
+  },
+//{[{
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+//}]}
```

Comment:
Just for learning purpose, what is this eslintConfig used for?

Reply:
it's used to configure ESLint. Since our react uses javascript it may be possible that our users are using ESLint or that they want to. This basically sets linting rules and such.

Reply:
More info here:
https://eslint.org/docs/user-guide/configuring